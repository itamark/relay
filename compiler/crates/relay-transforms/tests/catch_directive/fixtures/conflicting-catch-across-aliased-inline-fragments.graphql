fragment Foo on Node {
    ... on MaybeNodeInterface @alias(as: "aliased_fragment") {
        name @catch
    }
    name
}