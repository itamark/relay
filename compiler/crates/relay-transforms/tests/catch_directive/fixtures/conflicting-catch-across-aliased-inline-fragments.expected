==================================== INPUT ====================================
fragment Foo on Node {
    ... on MaybeNodeInterface @alias(as: "aliased_fragment") {
        name @catch
    }
    name
}
==================================== OUTPUT ===================================
fragment Foo on Node {
  ... on MaybeNodeInterface @__FragmentAliasMetadata
  # FragmentAliasMetadata {
  #     alias: WithLocation {
  #         location: conflicting-catch-across-aliased-inline-fragments.graphql:60:62,
  #         item: "aliased_fragment",
  #     },
  #     type_condition: Some(
  #         Interface(0),
  #     ),
  #     selection_type: Interface(0),
  # }
   {
    name @catch @__CatchMetadataDirective
    # CatchMetadataDirective {
    #     to: Result,
    #     path: "aliased_fragment.name",
    # }
    
  }
  name
}
